app/controllers/reducers_controller.rb

```diff
@@ -131,4 +137,15 @@ def reducer_params(klass)
   def record_not_valid(exception)
     render json: { error: exception.message }, status: 422
   end
+
+  def remove_unwanted_config_key(param_object)
```

Comment:
I find the name of this method a bit confusing. Maybe we can call it `reset_unneeded_reducer_keys(param_object)`?